lib/array_intersection.rb

```diff
@@ -1,6 +1,26 @@
+require "pry"
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: Time complexity is O(n) (or, O(n+m)) we are looping through one array O(n), then searching worst case for a look up a hash table O(n)
```

Comment:
I _suspect_ this is correct - but what is `m` here? 🙂

(Make sure to define _all_ the letters you use.)